### PR TITLE
Migrate icons.dart to null safety

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/devtools_app/lib/src/ui/icons.dart [hunk 1]
```diff
@@ -95,31 +95,31 @@ class CircleIcon extends StatelessWidget {
 }
 
 class CustomIconMaker {
-  final Map<String, Widget> iconCache = {};
+  final Map<String?, Widget> iconCache = {};
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/ui/icons.dart:98]: String should be non-nullable

**Review by kenzieschmoll — APPROVED** [review]
one comment - then lgtm

File: packages/devtools_app/lib/src/ui/icons.dart (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/ui/icons.dart:98]: done

**Comment by polina-c** [comment]
tested

**Pull request merged**